ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1,2 @@
-m_EditorVersion: 2018.4.6f1
+m_EditorVersion: 2019.1.13f1
```

Comment:
Please revert this change. This prevents the project from opening in Unity 2018.4. We have customers that require that version.

Thanks!